pkg/retention/space_default_lifecycle_release_retention_policy.go

```diff
@@ -0,0 +1,40 @@
+package retention
+
+import "github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/resources"
+
+type LifecycleReleaseRetentionPolicy struct {
+	QuantityToKeep int    `json:"QuantityToKeep"`
+	Strategy       string `json:"Strategy"`
+	Unit           string `json:"Unit"`
+	SpaceDefaultRetentionPolicy
+}
+
+func CountBasedLifecycleReleaseRetentionPolicy(quantityToKeep int, unit string, spaceId string, policyId string) *LifecycleReleaseRetentionPolicy {
```

Comment:
The convention that I can see in the code base for funcs that create new structs is `New*`

```suggestion
func NewCountBasedLifecycleReleaseRetentionPolicy(quantityToKeep int, unit string, spaceId string, policyId string) *LifecycleReleaseRetentionPolicy {
```